docs/src/prototypes/MenuButton/focusUtils.ts

```diff
@@ -0,0 +1,16 @@
+import { focusZoneUtilities, Menu } from '@stardust-ui/react'
+
+export const focusMenuItem = (menuRef: HTMLUListElement, order: 'first' | 'last') => {
+  const selector = `.${Menu.Item.slotClassNames.wrapper}:${order}-child .${Menu.Item.className}`
```

Comment:
yes, this is something that might strike a user, as `.${Menu.Item.className}:${order}-child` is the most intuitive thing to try